features/user_can_compose_message.feature

```diff
@@ -0,0 +1,20 @@
+Feature: User can compose message
+    As a user
+    In order to communicate with others
+    I should be able to send and receive messages
+
+    Background: 
+        Given the following user exists
+        | name | email          | password | password_confirmation |
+        | Greg | real@email.com | password | password              |
+        And I visit the site
+        When I click 'Login'
```

Comment:
Why are you testing compose message in the login feature?

Reply:
Just an error with branching.  Took a hot minute to find the problem but it's been deleted.